TexasHoldem.c

```diff
@@ -231,226 +223,19 @@ void DefineBet(int playerNum, double playerbets[])
 		
 		//four players no cpu
 		case 4:
-		printf("Player %d is the dealer, player %d is the small blind, and player %d is the big blind", dealer, (dealer+1)%4, (dealer+2)%4;
+		printf("Player %d is the dealer, player %d is the small blind, and player %d is the big blind", dealer, (dealer+1)%4, (dealer+2)%4);
 		printf("What is Player %d's bet? ", (dealer+3)%4);
 		scanf("%lf", playerbets[(dealer+3)%4]);
```

Comment:
don't forget the & for the scanf. I think this came up more than once in this function.